src/main/kotlin/com/programmersbox/fullmultiplatformcompose/BuilderParams.kt

```diff
@@ -28,7 +28,7 @@ class Android {
 }
 
 class IOS {
-    var appName: String by mutableStateOf("My Application")
+    var appName: String by mutableStateOf("CMP_With_KMP App")
```

Comment:
I don't think there needs to be any underlines here.

Reply:
sorry

I sent the wrong PR

I mainly want to change the versions of various libs (especially the kotlin part, which I want to upgrade to 2.0.20 or above)

I also added my commonly used repos and libs, and then sent the PR without distinguishing them.

Mainly I want to know if this plugin can be modified to support kotlin 2.0 or above and more lib and mvvm templates?